xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp

```diff
@@ -513,7 +513,7 @@ CAEChannelInfo CActiveAEDSP::GetInternalChannelLayout(AEStdChLayout stdLayout)
   return CAEUtil::GetAEChannelLayout(channelLayoutOut);
 }
 
-bool CActiveAEDSP::CreateDSPs(unsigned int &streamId, CActiveAEDSPProcessPtr &process, AEAudioFormat inputFormat, AEAudioFormat outputFormat, bool upmix,
+bool CActiveAEDSP::CreateDSPs(unsigned int &streamId, CActiveAEDSPProcessPtr &process, AEAudioFormat &inputFormat, AEAudioFormat &outputFormat, bool upmix,
```

Comment:
Yes, this changes the semantics of the code.
Moving from pass by value should be replaced with const ref to avoid accidental modifications